src/main/java/hudson/plugins/tfs/browsers/TeamSystemWebAccessBrowser.java

```diff
@@ -104,8 +104,14 @@ public String getDisplayName() {
         }
         
         public static String getBaseUrl(String urlExample) throws MalformedURLException {
-        	URL url = new URL(urlExample);
-        	return new URL(url.getProtocol(), url.getHost(), url.getPort(), String.format("/%s", FilenameUtils.getPath(url.getPath()))).toString();
+
+          URL url = new URL(urlExample);
+          String path = url.getPath();
+          // path needs to end with / and users sometimes forget so we add it
+          if (!path.endsWith("/")) {

```

Comment:
This is necessary to avoid user annoyance. There was a bug and there are earlier tests for this.